### PR TITLE
[FrameworkBundle] Allow .yaml file extension everywhere

### PR DESCRIPTION
**Pull request by ogizanagi**
| Q             | A
| ------------- | ---
| Branch?       | 3.4 
| Bug fix?      | no
| New feature?  | yes 
| BC breaks?    | no
| Deprecations? | no 
| Tests pass?   | yes
| Fixed tickets | #23205 
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -1177,7 +1177,10 @@ private function registerValidatorMapping(ContainerBuilder $container, array $co
         foreach ($container->getParameter('kernel.bundles_metadata') as $bundle) {
             $dirname = $bundle['path'];
 
-            if ($container->fileExists($file = $dirname.'/Resources/config/validation.yml', false)) {
+            if (
+                $container->fileExists($file = $dirname.'/Resources/config/validation.yml', false) ||
+                $container->fileExists($file = $dirname.'/Resources/config/validation.yaml', false)
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: what if both exists ?

Comment (ogizanagi) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: The `.yml` file will take precedence (actually it'll probably be better to de the reverse).
Do you think it's an issue?

**Review by fabpot — APPROVED** [review]
Having .yml first is more BC, so LGTM

**Comment by fabpot** [comment]
Thank you @ogizanagi.

**Pull request merged**